project/.travis.yml.twig

```diff
@@ -33,6 +33,7 @@ cache:
 
 env:
   global:
+    - COMPOSER_DISABLE_XDEBUG_WARN=1
```

Comment:
This is not supposed to be necessary. Composer is supposed to deactivate xdebug itself: https://getcomposer.org/doc/articles/troubleshooting.md#xdebug-impact-on-composer